src/main/java/io/appium/java_client/AppiumDriver.java

```diff
@@ -171,54 +181,6 @@ public AppiumDriver(URL remoteSessionAddress, String platformName, String automa
         setSessionId(sessionAddress.getId());
     }
 
-    /**
```

Comment:
these method were just moved to the end of the class definitions, no changes made in them